test/1.4/legacy/legacy_attributes.dml

```diff
@@ -0,0 +1,43 @@
+/*
+  © 2024 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+import "simics/simulator/conf-object.dml";
+
+saved int s;
+hook() h;
+
+attribute a is uint64_attr;
+
+bank b {
+    saved int s;
+    hook() h;
+    attribute a is uint64_attr;
+}
+
+method init() default {
+    local attr_value_t val = SIM_make_attr_uint64(0xC0FFEE);
```

Comment:
Note: we usually use lowercase hex. Unsure if the style mandates it, and for bogus test values I don't care.

Reply:
huh? we do? if so, we are **definitely** not consistent about it like, at all.